recipe/meta.yaml

```
@@ -1,37 +1,50 @@
 {% set name = "ogstools" %}
-{% set version = "0.0.3" %}
+{% set version = "0.4.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ogstools-{{ version }}.tar.gz
-  sha256: 367a515b0ba7266ff1a2b013b7767e5376ecf33035a378163c9d06d57cc34569
+  sha256: 55c2d56050140c18941d8c4cad219bd4f9c25226e2c51333247d43fa885c3a6c
 
 build:
-  entry_points:
-    - msh2vtu = ogstools.msh2vtu._cli:cli
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
   number: 0
 
 requirements:
   host:
-    - python >=3.9
+    - python >=3.10
     - setuptools
     - setuptools-scm
     - pip
   run:
-    - python >=3.9
+    - python >=3.10
+    - python-gmsh >=4.12.0
+    - h5py >=3.8.0
+    - jupytext >=1.14.0
+    - matplotlib-base >=3.7.1
     - meshio >=5.3.0
+    - pyvista >=0.40.1
+    - numpy >=1.20.0,<2.1
+    - scipy >=1.10.1
+    - pint >=0.22
+    - papermill >=2.4.0
+    - pandamesh >=0.2.1
+    - ogs >=6.5.3
+    - typeguard >=4.0.0
+    - lxml >=4.6.0
 
 test:
   imports:
     - ogstools
   commands:
     - pip check
```

If `ogs` is still not found maybe try 

```suggestion
    - pip list
    - pip check
```